### ISSUE TITLE
Thumbnails in OSD extension are not accessible via keyboard navigation

### ISSUE THREAD

**Issue by edsilv** [UniversalViewer/universalviewer#745]
**UV version:**
```
 universalviewer@3.x
```

**I'm submitting a:**

- [x] bug report
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**
Thumbnails in the left panel are not accessible via keyboard navigation.

**Expected behavior:**
Using the keyboard tab key should select each thumbnail in sequence.

**Comment by edsilv** [comment]
closing as duplicate of #750
